radio/src/gui/480x272/widgets/modelbmp.cpp

```diff
@@ -63,13 +63,24 @@ class ModelBitmapWidget: public Widget
       }
     }
 
+    uint8_t evalHash(const void * data, uint32_t size)
```

Comment:
I actually compared generated assy code (gcc-arm 4.7) and this looks faster, though mostly because `memcmp` doesn't seem to be very efficient.  If you wrote your own "memcmp" which just simply compared bytes by value it may be marginally faster, but we're talking 1-2 cycle(s) difference per loop iteration.

BTW the compiler inlined `evalHash()`.  The resulting `refresh()` is only 5 extra lines of assy over the old version, which also jumped to `memcmp` twice (`memcpy` gets inlined to 6 lines of assy each).  `memcmp` is 82 lines!

Just in case anyone was wondering.    :-)